### DISCUSSION TITLE
关于图片审核NsfwJs

### DISCUSSION THREAD

**Discussion by Xue-chuxin** [lsky-org/lsky-pro#491] [问答] [answered]
我自己使用docker搭建了一个NsfwJs

图片也可以正常上传

但是图片有违规内容不会标记为不健康

http://121.62.16.157:5000/?url=

是我配置的不对还是什么问题呢？
![QQ截图20220725135306](https://user-images.githubusercontent.com/69140474/180707818-5978e4db-1388-42d4-ad42-be09a94c7e5e.png)

**Comment by Xue-chuxin** [comment]
阈值调到了1也还是不行

**Comment by 0xxb** [comment, marked as answer]
https://docs.lsky.pro/docs/free/v2/group/picture-review.html#%E6%8E%A5%E5%8F%A3%E5%9C%B0%E5%9D%80
接口需要返回以下结构：
```
{
    "porn": 0,
    "sexy": 0,
    "hentai": 0,
    "neutral": 0,
    "drawing": 0
}
```
该接口应该提供一个表单属性用来上传文件，不应该是get qeury 参数 url

**Reply by Xue-chuxin** [reply]
好的 谢谢

**Comment by 0xxb** [comment]
类似这个项目：
https://github.com/arnidan/nsfw-api

**Comment by penndu** [comment]
https://dusays.com/495/

**Reply by Xue-chuxin** [reply]
好的 谢谢
